kuma/javascript/src/payments/components/error-message.jsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { gettext, Interpolated } from '../../l10n.js';
+
+const errorMessage = () => (
+    <Interpolated
+        id={gettext(
+            "We're sorry, something went wrong. Please contact <emailLink />."
+        )}
+        emailLink={
+            <a
+                target="_blank"
+                rel="noopener noreferrer"
+                href={`mailto:${window.mdn.contributionSupportEmail}`}
+            >
+                {gettext(window.mdn.contributionSupportEmail)}
```

Comment:
```suggestion
                {window.mdn.contributionSupportEmail}
```
because we don't want our localizers to localize an email address.